src/rpcnet.cpp

```diff
@@ -398,6 +399,34 @@ static UniValue GetNetworksInfo()
     return networks;
 }
 
+UniValue getdeprecationinfo(const UniValue& params, bool fHelp)
```

Comment:
Hmm... would this make more sense in `rpcmisc.cpp`? If I had to pick a specific category then I *would* put it in `net`, but I think it's closer to upstream's `getmemoryinfo`, in that it is about the local node. I'm not fussed about bikeshedding this though.

Reply:
I put it in `rpcnet.cpp` because it also returns version info, duplicating some of what `getnetworkinfo` does